packit_service/worker/handlers/github_handlers.py

```diff
@@ -141,6 +145,18 @@ class ProposeDownstreamHandler(JobHandler):
     triggers = [TheJobTriggerType.release]
     task_name = TaskName.propose_downstream
 
+    def __init__(
+        self,
+        package_config: PackageConfig,
+        job_config: JobConfig,
+        data: EventData,
+        task: Task,
+    ):
+        super().__init__(
+            package_config=package_config, job_config=job_config, data=data,
+        )
+        self.task = task
```

Comment:
What about handling the exceptions/retries on the task level to not need to introduce a new attribute here?

Reply:
This seems better to me as if the exception is called more then the RETRY_LIMIT, we want to do the handling like creating an issue and so and for that check of count we need again the celery Task